lib/router.js

```diff
@@ -78,4 +84,35 @@ Router.prototype._sanitizeMethod = function(method) {
   return sanitizedMethod;
 };
 
+/*
+  Simple path-to-regex translation based on the Express "string-based path" syntax
+*/
+Router.prototype._parseSimplePath = function(path) {
+  // Check for named placeholder crowding
+  if (/\:[a-zA-Z0-9]+\:[a-zA-Z0-9]+/g.test(path)) {
+    throw new Error('Invalid usage of named placeholders');
+  }
+
+  // Try parsing the string and converting special characters into regex
+  try {
+    // Parsing anonymous placeholders with simple backslash-escapes
+    path = path.replace(/(.|^)\*/g, function(m,escape) {
```

Comment:
What do you want to mean with `/(.|^)/`. Are you sure you're trying to say "[any character or `^` character](https://regex101.com/r/aA2gY3/1)"?